server/src/optima/dbmodels.py

```diff
@@ -46,7 +46,9 @@ class ProjectDb(db.Model):
                                 uselist=False)
     project_data = db.relationship('ProjectDataDb', backref='projects',
                                 uselist=False)
-    creation_time = db.Column(db.DateTime(timezone=True), server_default=text('now()'))
+    creation_time = db.Column(db.DateTime(timezone=True), default=db.func.now())
```

Comment:
The change on `creation_time` is for consistency reasons, and also I think using `default` instead of `server_default` is more robust because it allows defaults on INSERT and UPDATE. Only think we could add is one line in the migration script, to align the database:
```sql
ALTER COLUMN creation_time DROP DEFAULT;
```
For `updated_time` the default is not problematic, and can be removed, but still it's not source of confusion because same `creation_time` and `updated_time` means no update.